.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]*"
```

Comment:
Could we verify that in-script version was bumped as well? (I guess it should match the tag)